### PR TITLE
[zh-cn] resync: proxies.md

### PR DESCRIPTION
**Pull request by hs0210**
Signed-off-by: Hu Shuai <hus.fnst@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/concepts/architecture/garbage-collection.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ allows the clean up of resources like the following:
 垃圾收集允许系统清理如下资源：
 
 <!--
-* [Failed pods](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection)
+* [Terminated pods](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection)
 * [Completed Jobs](/docs/concepts/workloads/controllers/ttlafterfinished/)
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/architecture/garbage-collection.md:24]: There is another PR already working on it.
- #35787
(Made by the en PR author, and 8 hours before this PR open)

**Review by Sea-n — COMMENTED** [review]
Hold until conflicted PR merged and rebase this one.
/hold

File: content/zh-cn/docs/concepts/architecture/garbage-collection.md (hunk 1 above)
Comment (hs0210) [reply content/zh-cn/docs/concepts/architecture/garbage-collection.md]: > Hold until conflicted PR merged and rebase this one.

Thanks. I have rebased it, PTAL.

**Comment by mengjiao-liu** [comment]
I see the en file has been updated. Please fetch the newest main branch and run the following command.
```
./scripts/lsync.sh content/zh-cn/docs/concepts/cluster-administration/proxies.md
```

**Comment by hs0210** [comment]
> I see the en file has been updated. Please fetch the newest main branch and run the following command.

@mengjiao-liu Done. PTAL.

**Comment by mengjiao-liu** [comment]
/approve
/lgtm
Thanks!

**Pull request merged**